scripts/find_spammy_content.py

```diff
@@ -70,3 +72,25 @@ def find_spammy_content(regex, days, model, ban):
             if ban:
                 item.suspend_spam_user(item.creator, train_akismet=False)
     return data
+
+def find_spammy_content_fast(regex, days, model, ban):
+    spam = model.objects.filter(
+        (Q(title__iregex=regex)|Q(description__iregex=regex)) &
+        Q(created__gte=timezone.now() - timedelta(days=days)) &
+        ~Q(spam_status=2) &
+        Q(deleted__isnull=True)
+    )
+    data = []
+    for item in spam.values('guids___id', 'title', 'description', 'created', 'creator__guids___id', 'creator__username', 'creator__fullname'):
+        item_data = {}
+        item_data['type'] = model.__name__
+        item_data['guid'] = item['guids___id']
+        item_data['content'] = item['title'] + '::' + item['description']
+        item_data['created'] = item['created']
+        item_data['uid'] = item['creator__guids___id']
+        item_data['username'] = item['creator__username']
+        item_data['fullname'] = item['creator__fullname']
+        data.append(item_data)
+        if ban:
+            model.load(item['guids___id']).suspend_spam_user(item['creator__guids___id'], train_akismet=False)
```

Comment:
Not ideal, but `suspend_spam_user` requires an instance of `Model(SpamOverrideMixin)`